uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-report.scss

```diff
@@ -8,6 +8,18 @@
       background: #e6e6e6;
     }
   }
+
+  table {
+    table-layout: fixed;
+
+    thead th,
+    tbody td {
+      overflow:hidden;
+      padding: 8px 5px;
+      white-space: normal !important;
```

Comment:
why `!important`? can it be avoided? (if not, that's ok)

Reply:
I double checked, and this whole css style is not required.